tests/perf-test/src/main/java/com/paritytrading/philadelphia/perf/FIXValueBenchmark.java

```diff
@@ -73,8 +75,10 @@ public void prepare() {
         timeOnlyValue = new FIXValue(32);
         timeOnlyValue.setTimeOnly(timeOnly, true);
 
-        timestampValue = new FIXValue(32);
+        timestampValue = new FIXValue(64);
```

Comment:
Why the increase in the value capacity?

Reply:
The default `FIXValue` is actually 64 via `FIXConfig`, and as it's a perf test I thought to set it to the default.

Reply:
Good point. :+1: Maybe we could the same for all `FIXValue` instances in this benchmark? :wink: